zuul/layout.yaml

```diff
@@ -67,6 +67,36 @@ pipelines:
     ignore-dependencies: False
 
 
+  - name: check-vgpu-xenserver
```

Comment:
Does this imply its a new pipeline?
Our CI can only vote once - so we need to merge the jobs needed into one pipeline rather than having separate pipelines per job.

Reply:
Yes, it is a new pipeline. According to Jianghua's suggestion, we don't want the request event for vgpu to be put into the same queue with other requests(Because we used a specified host for vgpu), so I add a new pipeline for vgpu.
It looks Zuul would wait all the pipeline finished and give a vote according to the results

Reply:
I think that's fine to use a separate pipeline. We have separate pipeline for check-coverage.

Reply:
OK - thanks for the reassurance

Reply:
Very sorry. I was wrong. We need put this vGPU test in the pipeline of "check". The check-coverage is in check pipeline also.